scripts/docker/ubuntu.14.04/Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM ubuntu:14.04
+
+# Misc Dependencies for build
+RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+
+# This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
+RUN echo "deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
+    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
+    apt-get update && apt-get -qqy install\
+	libc6 \
```

Comment:
To determine this list @ellismg had an awesome bash one liner that ran `ldd` against the shared libraries distributed by corefx and coreclr to determine hte entire package closure. 

Undoubtedly some of these packages are contained as dependenices of others, but it doesn't seem straightforward to determine which. So I'd say this is similar to the lock file of our package dependencies on ubuntu 14.04.